cli/cache/common.rs

```diff
@@ -3,16 +3,17 @@
 use std::hash::Hasher;
 
 /// A very fast insecure hasher that uses the xxHash algorithm.
-#[derive(Default)]
```

Comment:
Now forces someone to decide whether to include the deno version in the cache or not.